.github/workflows/inactivity_reminder.yaml

```diff
@@ -0,0 +1,77 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Inactivity Reminder with Different Times
+
+on:
+  schedule:
+    - cron: '0 9 * * *'  # Runs daily at 09:00 UTC
```

Comment:
I think it'd be helpful to add `workflow_dispatch:` for these workflows, so you can test changes by manually triggering them from the GitHub UI.

That'd only work if you opened the PR from a `cuopt` branch (not your fork), though.

Reply:
@jameslamb tested it and behavior is noted in the description https://github.com/NVIDIA/cuopt/pull/268#issue-3305082279